### ISSUE TITLE
Can't switch sqlite to mariadb after I install nextcloud

### ISSUE THREAD

**Issue by yuzhen8** [nextcloud/docker#1480]
I used Apache and MariaDB separately. but when I login nextcloud gui in first. it's note me you has seleced sqlite as database. In fact I has add --link and nextcloud recommonded environment variables in docker command. Following is my docker command.

```
**1. create datebase** 
docker run -d --name my-mysql -p 0.0.0.0:3306:3306 -v db:/var/lib/mysql -e MYSQL_ROOT_PASSWORD=nextcloud -e MYSQL_PASSWORD=nextcloud -e MYSQL_DATABASE=mysql -e MYSQL_USER=nextcloud  mariadb:latest

**2.create apache&php**
docker run -itd --name apache-php-nextcloud -p 8081:80 -v nextcloud:/var/www/html --link my-mysql:mysql -e MYSQL_HOST=mysql -e MYSQL_PASSWORD=nextcloud -e MYSQL_DATABASE=mysql -e MYSQL_USER=nextcloud php7.3-apache:1.0

```

**Comment by rashkov** [comment]
you might need to delete your volumes and run the commands again

**Comment by yuzhen8** [comment]
> you might need to delete your volumes and run the commands again

you mean I need delete volume after remove container everytime?

**Comment by rashkov** [comment]
No, just this once.. assuming that you have no valuable data in your volumes currently.

**Comment by yuzhen8** [comment]
> No, just this once.. assuming that you have no valuable data in your volumes currently.

Yes, In fact, I'm sure the volume no valuable data. I has done unzip "nextcloud.zip" after create container.

**Comment by hrj5032188** [comment]
I had the same problem

On the index page, click Install complete

Page prompt me [MySQL数据库用户名和/或密码无效 您需要输入现有账号的详细信息。]

![image](https://user-images.githubusercontent.com/28888068/125161175-6d0e9580-e1b3-11eb-8974-08e68c189180.png)

But nextcloud has created users and database in MySQL

![image](https://user-images.githubusercontent.com/28888068/125161198-844d8300-e1b3-11eb-98f1-1801df7e13a1.png)
